### ISSUE TITLE
Component placement diagram

### ISSUE THREAD

**Issue by Pedro147** [AlienFlight/Flight-Controllers#2]
I have found that a diagram, like the one attached can be very useful when it comes time to assembling the PCB's prior to reflow, and am just wondering if this is something that might be worth making available. 
This is one that I did for a related project of my own and is at this stage only an incomplete first draft, but if deemed appropriate, I could do this style of diagram for all the available boards. I found it an invaluable aid in ensuring that the correct component is placed in the correct position.
![parts placement](https://cloud.githubusercontent.com/assets/5655134/12986889/9ede8998-d14d-11e5-81ac-b78614a0e55b.jpg)

**Comment by HumbleOne** [comment]
@Pedro147, If I understand correctly, you're offering to make part/value placement diagrams available? If so, let's restart by using example board material directly from this project. :wink:

**Comment by Pedro147** [comment]
@HumbleOne yes you are correct and I only offered this as an example of what I was proposing seeing as I already had this done for another project.:)

It seemed pointless to do one for the current AlienFlight boards if there was no interest in the general concept. I will do one for the  Classic-Narrow-5V-SIDE-USB and submit that

**Comment by HumbleOne** [comment]
@Pedro147, this a good idea, and I can clearly see it's practical application.
With that, I say go ahead, let's see how it turns out. :+1: 

I feel you are already taking a nice direction by boxing in each part/value. 

Alternately, since the values are already there in the board file, it would take less work to make a copy of the board file and, if necessary just edit the value's size and placement. Maybe even add layer 17 and 21, 51 back in as those should provide a good visual of the components. 

Ultimately, It's up to you for the direction you wish to take. I do like your idea of creating containers too.

**Comment by Pedro147** [comment]
@HumbleOne  Thanks for the positive suggestions, I appreciate your input :)

I will have a play around with the possibilities and post my proposal back here :alien:
